python/oneflow/nn/graph/graph_config.py

```diff
@@ -56,48 +56,48 @@ def set_outputs_buffer_size(self, value: int = 2):
         self._outputs_buffer_size = value
 
     def enable_amp(self, mode: bool = True):
-        """If true, then graph will use mixed precision mode, it means use both float16 and float32 during model training.
+        r"""If true, then graph will use mixed precision mode, it means use both float16 and float32 during model training.
```

Comment:
```suggestion
        r"""Enable mixed precision mode or not.
        
        Args:
            mode(bool): Set to true to use mixed precision mode, otherwise set to false. The mixed precision mode means to use both float16 and float32 during model training.
```

我觉得这里一系列 “if true ..." 开头的 docstring，语言都比较奇怪，我找了个 MSDN 中的文档例子，试着改了下。就是把一部分信息，放到 Args 下去了。

即使 Args 的信息保持现状不改，我还是建议 "If true" 这类文字要改一下，有点怪怪的。

```
[ToolTip.setActivate]()

Activates or deactivates the ToolTip control.

Syntax

public void setActivate( boolean value )

Parameters

value

Set to true to activate the control; otherwise, set to false.
```

Reply:
好的 我参考一下

Reply:
是的，把 if true 改为 set to true 就很好，是 it is set to true 的简写；也可 if set to true